tests/Functional/tests/enumerated-ignored/enumerated-ignored.rst

```diff
@@ -0,0 +1,6 @@
+The second line of each enumerated list item is checked for validity:
+
+1. This is not a list.
+It's a paragraph starting with a number.
```

Comment:
Previously, there was a tests that tested this behavior as "being recognized as a list" (`tests/Functional/tests/ordered2/ordered2.rst`). However, [the markup specs](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#enumerated-lists) are very explicit about this:

> The second line of each enumerated list item is checked for validity. This is to prevent ordinary paragraphs from being mistakenly interpreted as list items, when they happen to begin with text identical to enumerators. For example, this text is parsed as an ordinary paragraph:
> 
> ```
> A. Einstein was a really
> smart dude.
> ```
> 
> However, ambiguity cannot be avoided if the paragraph consists of only one line. This text is parsed as an enumerated list item:
> 
> `A. Einstein was a really smart dude.`
> 
> If a single-line paragraph begins with text identical to an enumerator ("A.", "1.", "(b)", "I)", etc.), the first character will have to be escaped in order to have the line parsed as an ordinary paragraph:
> 
> `\A. Einstein was a really smart dude.`